packages/esm-patient-tests-app/src/test-results/panel-view/panel.component.test.tsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import LabSetPanel from './panel.component';
+import { isDesktop, useLayoutType, type LayoutType } from '@openmrs/esm-framework';
+import { type ObsRecord } from '../../types';
+import { type OBSERVATION_INTERPRETATION } from '@openmrs/esm-patient-common-lib';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  formatDate: jest.fn().mockReturnValue('January 1, 2024'),
+  isDesktop: jest.fn().mockReturnValue(true),
+  useLayoutType: jest.fn().mockReturnValue('desktop'),
+}));
```

Comment:
```suggestion
```

We don't need to override the partial framework mock here https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#you-should-not-need-to-use-partial-mocks.